The failures don't look related:

```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.262 s <<< FAILURE! - in io.quarkus.it.main.TransactionalTestMethodTestCase
[ERROR] testUserTransaction  Time elapsed: 0.016 s  <<< ERROR!
org.junit.jupiter.api.extension.TestInstantiationException: TestInstanceFactory [io.quarkus.test.junit.QuarkusTestExtension] failed to instantiate test class [io.quarkus.it.main.TransactionalTestMethodTestCase]: Failed to create the boostrap class loader
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeTestInstanceFactory(ClassBasedTestDescriptor.java:300)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateTestClass(ClassBasedTestDescriptor.java:275)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.instantiateTestClass(ClassTestDescriptor.java:77)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateAndPostProcessTestInstance(ClassBasedTestDescriptor.java:258)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$2(ClassBasedTestDescriptor.java:252)
	at java.util.Optional.orElseGet(Optional.java:267)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$3(ClassBasedTestDescriptor.java:251)
	at org.junit.jupiter.engine.execution.TestInstancesProvider.getTestInstances(TestInstancesProvider.java:29)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$prepare$0(TestMethodTestDescriptor.java:106)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:105)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:69)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$prepare$1(NodeTestTask.java:107)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.prepare(NodeTestTask.java:107)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:75)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:142)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:117)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.IllegalStateException: Failed to create the boostrap class loader
	at io.quarkus.test.junit.QuarkusTestExtension.createQuarkusBuildClassLoader(QuarkusTestExtension.java:334)
	at io.quarkus.test.junit.QuarkusTestExtension.doJavaStart(QuarkusTestExtension.java:92)
	at io.quarkus.test.junit.QuarkusTestExtension.createTestInstance(QuarkusTestExtension.java:393)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeTestInstanceFactory(ClassBasedTestDescriptor.java:285)
	... 49 more
Caused by: io.quarkus.bootstrap.BootstrapException: Failed to create the deployment classloader for io.quarkus:quarkus-integration-test-main::jar:999-SNAPSHOT
	at io.quarkus.bootstrap.BootstrapClassLoaderFactory.newDeploymentClassLoader(BootstrapClassLoaderFactory.java:329)
	at io.quarkus.test.junit.QuarkusTestExtension.createQuarkusBuildClassLoader(QuarkusTestExtension.java:332)
	... 52 more
Caused by: io.quarkus.bootstrap.resolver.AppModelResolverException: Failed to resolve artifacts
	at io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver.resolve(MavenArtifactResolver.java:188)
	at io.quarkus.bootstrap.resolver.BootstrapAppModelResolver.doResolveModel(BootstrapAppModelResolver.java:194)
	at io.quarkus.bootstrap.resolver.BootstrapAppModelResolver.resolveManagedModel(BootstrapAppModelResolver.java:133)
	at io.quarkus.bootstrap.resolver.BootstrapAppModelResolver.resolveModel(BootstrapAppModelResolver.java:124)
	at io.quarkus.bootstrap.BootstrapClassLoaderFactory.newDeploymentClassLoader(BootstrapClassLoaderFactory.java:314)
	... 53 more
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact io.quarkus:quarkus-security-spi:jar:999-SNAPSHOT
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:422)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:224)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveArtifacts(DefaultRepositorySystem.java:268)
	at io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver.resolve(MavenArtifactResolver.java:186)
	... 57 more
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact io.quarkus:quarkus-security-spi:jar:999-SNAPSHOT
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:412)
	... 60 more


[ERROR] Errors: 
[ERROR]   TransactionalTestMethodTestCase.testUserTransaction » TestInstantiation TestIn...
[INFO] 
[ERROR] Tests run: 39, Failures: 0, Errors: 1, Skipped: 38
[INFO] 
```

--- a/test-framework/maven/src/main/java/io/quarkus/maven/it/verifier/RunningInvoker.java
+++ b/test-framework/maven/src/main/java/io/quarkus/maven/it/verifier/RunningInvoker.java
@@ -1,13 +1,25 @@
 package io.quarkus.maven.it.verifier;
 
-import java.io.*;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.file.Files;
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 import java.util.stream.Collectors;
 
 import org.apache.commons.io.FileUtils;
-import org.apache.maven.shared.invoker.*;
+import org.apache.commons.io.output.TeeOutputStream;
+import org.apache.maven.shared.invoker.DefaultInvocationRequest;
+import org.apache.maven.shared.invoker.InvocationRequest;
+import org.apache.maven.shared.invoker.InvocationResult;
+import org.apache.maven.shared.invoker.InvokerLogger;
+import org.apache.maven.shared.invoker.MavenInvocationException;
+import org.apache.maven.shared.invoker.PrintStreamHandler;
+import org.apache.maven.shared.invoker.PrintStreamLogger;
 import org.jutils.jprocesses.JProcesses;
 import org.jutils.jprocesses.model.ProcessInfo;
 
@@ -20,9 +32,10 @@ public class RunningInvoker extends MavenProcessInvoker {
     private final boolean debug;
     private MavenProcessInvocationResult result;
     private final File log;
-    private final PrintStreamHandler logHandler;
+    private final PrintStreamHandler outStreamHandler;
+    private final PrintStreamHandler errStreamHandler;
 
-    public RunningInvoker(File basedir, boolean debug) throws FileNotFoundException {
+    public RunningInvoker(File basedir, boolean debug) {
         this.debug = debug;
         setWorkingDirectory(basedir);
         String repo = System.getProperty("maven.repo");
@@ -31,16 +44,44 @@ public class RunningInvoker extends MavenProcessInvoker {
         }
         setLocalRepositoryDirectory(new File(repo));
         log = new File(basedir, "build-" + basedir.getName() + ".log");
-        PrintStream stream = null;
+        PrintStream outStream;
         try {
-            stream = new PrintStream(log, "UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            stream = new PrintStream(log);
+            outStream = createTeePrintStream(System.out, Files.newOutputStream(log.toPath()));
+        } catch (IOException ioe) {
+            outStream = System.out;
         }
-        logHandler = new PrintStreamHandler(stream, true);
-        setErrorHandler(logHandler);
-        setOutputHandler(logHandler);
-        setLogger(new PrintStreamLogger(stream, InvokerLogger.DEBUG));
+        this.outStreamHandler = new PrintStreamHandler(outStream, true);
+        setOutputHandler(this.outStreamHandler);
+
+        PrintStream errStream;
+        try {
+            errStream = createTeePrintStream(System.err, Files.newOutputStream(log.toPath()));
+        } catch (IOException ioe) {
+            errStream = System.err;
+        }
+        this.errStreamHandler = new PrintStreamHandler(errStream, true);
+        setErrorHandler(this.errStreamHandler);
+
+        setLogger(new PrintStreamLogger(outStream, debug ? InvokerLogger.DEBUG : InvokerLogger.INFO));
+    }
+
+    /**
+     * Creates a {@link PrintStream} with an underlying {@link TeeOutputStream} composed of {@code one}
+     * and {@code two} outputstreams
+     *
+     * @param one
+     * @param two
+     * @return
+     */
+    private static PrintStream createTeePrintStream(final OutputStream one, final OutputStream two) {
+        final OutputStream tee = new TeeOutputStream(one, two);
+        PrintStream stream;
+        try {
+            stream = new PrintStream(tee, true, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            stream = new PrintStream(tee, true);
+        }
+        return stream;
     }
 
     public void stop() {
@@ -81,8 +122,8 @@ public class RunningInvoker extends MavenProcessInvoker {
 
         request.setShellEnvironmentInherited(true);
         envVars.forEach(request::addShellEnvironment);
-        request.setOutputHandler(logHandler);
-        request.setErrorHandler(logHandler);
+        request.setOutputHandler(outStreamHandler);
+        request.setErrorHandler(errStreamHandler);
         this.result = (MavenProcessInvocationResult) execute(request);
         return result;
     }
@@ -93,6 +134,9 @@ public class RunningInvoker extends MavenProcessInvoker {
     }
 
     public String log() throws IOException {
+        if (log == null) {
+            return null;
+        }
         return FileUtils.readFileToString(log, "UTF-8");
     }
 

